.github/workflows/rust-minimal.yml

```diff
@@ -57,8 +67,11 @@ jobs:
       with:
         components: clippy, rustfmt
 
+    # Colcon can not be run in a venv which is required in Ubuntu Noble
+    # Removing the externally managed file 
     - name: Install colcon-cargo and colcon-ros-cargo
       run: |
+        sudo rm -f /usr/lib/python3.12/EXTERNALLY-MANAGED
```

Comment:
Very interesting, I guess this is an alternative to the `--break-system-packages` flag? I didn't know this could be done!

Reply:
ah yes I read about that flag too. But I got this idea from my co-maintainer at Crazyswarm2 (see this [line](https://github.com/IMRCLab/crazyswarm2/blob/c08db30a187858de16cbea7fa387ebe0370b5dd0/.github/workflows/ci-ros2.yml#L47)). But seems to work out!